lang/src/error.rs

```diff
@@ -15,13 +15,13 @@ pub enum ErrorCode {
 
     // IDL instructions.
     #[msg("The program was compiled without idl instructions")]
-    IdlInstructionStub = 120,
+    IdlInstructionStub = 1000,
```

Comment:
Why bump the error codes?

Reply:
because the intervals between some of them were too small. So I bumped all of them for consistency

Reply:
We'll need a `# Breaking` entry in the CHANGELOG for this.